data/json/monsters/zed_soldiers.json

```diff
@@ -62,7 +62,35 @@
     "type": "MONSTER",
     "name": { "str": "zombie soldier test only" },
     "description": "This zombie soldier is for testing purposes only.",
-    "copy-from": "mon_zombie_soldier"
+    "copy-from": "mon_zombie_soldier",
```

Comment:
It'll be easier to just override the whole field {"weakpoints": []}

Support for (Extend / Delete) isn't fully implemented. I wasn't sure about the best format (only specify the name / specify the entire weakpoint).